andela/ah-kgl-avengers-frontend
# #165020159 Setting up react-router-v5

### What does this PR do?
- use react-router v5 to access different routes
### Description of Task to be completed?
Configuring react router 5 in the project.
fixes the bug in `webpack.config.babel.js` that was causing the `.env.sample` file error complaint
### How should this be manually tested?
- `git clone https://github.com/andela/ah-kgl-avengers-frontend.git`
- git checkout to `ch-Setup-react-router-v5-165020159`
- npm i

 - Check out to these routes
 - `localhost:8000`
 - `localhost:8000/sign-up`
 - `localhost:8000/about`
### Any background context you want to provide?
- This PR also adds budges to the readme
### What are the relevant pivotal tracker stories?
[#165020159](https://www.pivotaltracker.com/story/show/165020159) Setup react router version 5 (React-router-dom)


### Screenshots (if appropriate)
```javascript
changed 
.env.<sample>

to 

.env.<example>
```
![image](https://user-images.githubusercontent.com/33352484/59020071-8660ed00-8849-11e9-8735-729386082633.png)



### Questions:
